plugins/SitesManager/API.php

```diff
@@ -1346,6 +1357,43 @@ public function setDefaultTimezone($defaultTimezone)
         return true;
     }
 
+    public function setExclusionTypeForQueryParams(string $exclusionTypeForQueryParams): void
```

Comment:
Having extra API for setting the type and setting the parameters to me feels a bit unhandy.
Currently when using the `setGlobalExcludedQueryParameters` API, that would only have an effect if the type is set correctly 🤔 
Would it make more sense to have one API, to set type and (optional) the exclusion list? The old API methods could still exist for BC, but should automatically set the correct type when called.

Reply:
@sgiehl 

Yeah I agree, also to retain some form of data integrity so we can't get into an invalid state. I've updated this now.

Would you say I then need to deprecate the other `set` method?